src/index.js

```diff
@@ -0,0 +1,74 @@
+export default function multiDeck() {
+    const dice6 = [1, 2, 3, 4, 5, 6];
+    const numberOfDecks = 5; //<-- This is how many times the deck gets repeated
+    let cards = [];
+
+    for (let die1Index = 0; die1Index < dice6.length; die1Index++) {
+        for (let die2Index = 0; die2Index < dice6.length; die2Index++) {
+            let die1 = dice6[die1Index];
+            let die2 = dice6[die2Index];
+            let sum;
+            try {
+                sum = sumDice(die1, die2);
+            } catch (e) {
+                console.error(e);
+            }
+            let card = {
+                die1,
+                die2,
+                sum
+            };
+            cards.push(card);
+        }
+    }
+
+    // Repeat the deck n number of times
+    const repeatedDeck = repeatDeck(cards, numberOfDecks);
+
+    // Shuffle the repeated deck to randomise order.
+    const shuffledRepeatedDeck = shuffleArray(repeatedDeck);
+
+    //Return (output) the shuffled deck
+    return shuffledRepeatedDeck;
```

Comment:
Set function as an inline export:
export function shuffleArray(array) {

Reply:
I've created a shouldShuffle arg and set the default to true. 

Because there are 2 default args, if only shouldShuffle is passed it wont use the default number of decks as it is expecting 2 args. So I have added this code that checks if only 1 arg is passed, if it is 'not a number' then it sets the variables as needed. 

```
    if (typeof numberOfDecks !== 'number') {
        shouldShuffle = numberOfDecks; // Shift arguments if only one provided
        numberOfDecks = 5; // Set default value for numberOfDecks
    }
```